index.ts

```diff
@@ -58,40 +59,40 @@ export async function isRunning(player: PlayerName) {
     }
 }
 
-/**
+/** review name
  * Kills the Spotify desktop if it's running
  */
 export function stopSpotifyIfRunning() {
     return musicCtr.stopPlayer(PlayerName.SpotifyDesktop);
 }
 
-/**
+/** review name
  * Kills the iTunes desktop if it's running
  */
 export function stopItunesIfRunning() {
```

Comment:
I think "stopItunes" is enough ... (whether it was running or not could be part of a return value???)